vocabularies/CodeList.xml

```diff
@@ -70,6 +70,16 @@
         </Annotation>
       </Term>
 
+      <Term Name="DoNotCheckScale" Type="Edm.Boolean" Nullable="false" AppliesTo="Property">
```

Comment:
```suggestion
      <Term Name="DoNotCheckScale" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property">
```
Why is this not a tag?

Reply:
What value does that indirection add?

Reply:
Don't know why Core.Tag was introduced but it is used in similar cases. If it does not add value, why was it introduced?